buildlib/tools/builds.sh

```diff
@@ -98,7 +98,7 @@ build_release_pkg() {
 	# check that UCX version is present in spec file
 	cd ${WORKSPACE}
 	# extract version from configure.ac and convert to MAJOR.MINOR.PATCH representation
-	version=$(grep -P "define\S+ucx_ver" configure.ac | awk '{print $2}' | sed 's,),,' | xargs echo | tr ' ' '.')
+	version=$(grep -P "define\S+ucx_ver" configure.ac | awk '{print $2}' | sed 's/\.\|)//g' | xargs echo | tr ' ' '.')
```

Comment:
IMO we should grep for ucx_ver_extra/major/minor explicitly and build the version string to compare to. also, update the comment the line above.